tools/static-assets/skel/test/main.js

```diff
@@ -1,27 +1,20 @@
 import assert from "assert";
 
-export const testMessage = "Welcome to Meteor!";
-
-describe("~name~", () => {
-  it("package.json has correct name", () => {
-    const { name } = require("../package.json");
+describe("~name~", function () {
+  it("package.json has correct name", async function () {
+    const { name } = await import("../package.json");
```

Comment:
I realized this `package.json` test could be merged with the dynamic `import()` test below, so we don't have to call `require` at all, and we can show off dynamic `import()` more prominently.